.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: python
+python:
+  # We don't actually use the Travis Python, but this keeps it organized.
+  - "2.7"
+  - "3.4"
+before_install:
+  - sudo apt-get update -qq
+install:
+  # You may want to periodically update this, although the conda update
+  # conda line below will keep everything up-to-date.  We do this
+  # conditionally because it saves us some downloading if the version is
+  # the same.
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    fi
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no 
+  - conda config --add channels pyne
+  - conda config --add channels cyclus
+  - conda update -q conda
+  - conda install conda-build jinja2 setuptools binstar patchelf nose
```

Comment:
It is:
https://github.com/opotowsky/cymetric/blob/trav/conda-recipe/meta.yaml#L27

On Thu, May 7, 2015 at 6:56 PM, Anthony Scopatz <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/opotowsky/cymetric/pull/2#discussion_r29907306>:
>
> > +  # conda line below will keep everything up-to-date.  We do this
> > +  # conditionally because it saves us some downloading if the version is
> > +  # the same.
> > +  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
> > +      wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
> > +    else
> > +      wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
> > +    fi
> > +  - bash miniconda.sh -b -p $HOME/miniconda
> > +  - export PATH="$HOME/miniconda/bin:$PATH"
> > +  - hash -r
> > +  - conda config --set always_yes yes --set changeps1 no
> > +  - conda config --add channels pyne
> > +  - conda config --add channels cyclus
> > +  - conda update -q conda
> > +  - conda install conda-build jinja2 setuptools binstar patchelf nose
>
> glib should be part of the dependencies listed in meta.yml.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opotowsky/cymetric/pull/2/files#r29907306>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192